scripts/quantos-dosing-series/example.yaml

```diff
@@ -0,0 +1,9 @@
+compound1_name : DMSO
+compound2_name : H2O
+n_fractions : 11
+total_mass : 12
```

Comment:
Are comments tolerated?  If so, might be good to note `# grams` after `total_mass` and `# grams/mol` after `mw`